scripts/hotspot.py

```diff
@@ -20,13 +20,13 @@ def parse_command():
     advanced_options = parser.add_argument_group('advanced user arguments')
     hotspot_options.add_argument('--genomic_hotspots',
         nargs='*',
-        help="Tab-delimited BED formatted file(s) containing hg38 genomic positions corresponding to hotspots")
+        help="Tab-delimited BED-formatted (chrom, zero-based chromStart, non-inclusive chromEnd) file(s) containing hg38 genomic positions corresponding to hotspots")
     hotspot_options.add_argument('--protein_snvs',
         nargs='*',
-        help="Column-name-containing, tab-delimited file(s) containing protein names and position for SNV hotspots")
+        help="Column name-labeled, tab-delimited file(s) containing HUGO-formatted protein names and VEP-formatted positions <start_aa_pos>-<end_aa_pos> for SNV hotspots")
```

Comment:
Hmmm, SNVs won't have a start and end though, a person might end up creating something like:

Hugo_Symbol | Amino_Acid_Position
-- | --
ACVR1 | 328-328

Is that ok?

Reply:
Maybe I should just put the regex in there

Reply:
That being said if they did put `328-328` it would work too.

Reply:
Fixed